sdk/log/provider.go

```diff
@@ -254,3 +257,21 @@ func WithAttributeValueLengthLimit(limit int) LoggerProviderOption {
 		return cfg
 	})
 }
+
+// AllowKeyDuplication sets whether deduplication is skipped for log attributes or other key-value collections.
+//
+// By default, the key-value collections within a log record are deduplicated to comply with the OpenTelemetry Specification,
```

Comment:
Nit, but this comma (in addition to the uppercase afterwards) is a bit weird here.


```suggestion
// By default, the key-value collections within a log record are deduplicated to comply with the OpenTelemetry Specification.
```